### PR TITLE
Modified the normalizer to handle single values properly + Neo4j Writer Logs URI

### PR DESCRIPTION
**Pull request by angelosantos4**
- Fixing a bug with the normalize value provider that crashes when the data returns None. 
- Adding database uri for logging. This can be used to specify logs between different database targets.
- Increase the timeout window for error collecting tests for more deterministic tests.

### CODE REVIEW

File: nodestream/pipeline/value_providers/normalizer_value_provider.py [hunk 1]
```diff
@@ -13,7 +13,11 @@ def __init__(self, using: str, data: ValueProvider):
         self.data = data
 
     def single_value(self, context: ProviderContext) -> Any:
-        return next(self.many_values(context))
+        values = list(self.many_values(context))
+        if values:
+            return next((value for value in values))
+        else:
+            return None
```
Comment (zprobst) [inline nodestream/pipeline/value_providers/normalizer_value_provider.py:20]:
```suggestion
        return next(self.many_values(context), default=None)
```

I think this is a cleaner solution

File: nodestream/databases/neo4j/query_executor.py [hunk 2]
```diff
@@ -25,6 +25,7 @@ def __init__(
         chunk_size: int = 1000,
         execute_chunks_in_parallel: bool = True,
         retries_per_chunk: int = 3,
+        alias: str = "neo4j-default",
```
Comment (zprobst) [inline nodestream/databases/neo4j/query_executor.py:28]: Why introduce another value here? Why not rely on the URI and log it out?  See [here](https://github.com/nodestream-proj/nodestream/blob/280298babb589743aa544074a3bd6a45cdf44001/nodestream/databases/neo4j/query_executor.py#L94).

Comment (angelosantos4) [reply nodestream/databases/neo4j/query_executor.py:28]: Figured we wouldn't want the uri in the logs, but I can add it into the log message instead.

File: nodestream/pipeline/value_providers/normalizer_value_provider.py (hunk 1 above)
Comment (angelosantos4) [reply nodestream/pipeline/value_providers/normalizer_value_provider.py:20]: Yeah thats way cleaner didn't even notice that was an option.

**Review by zprobst — APPROVED** [review]

**Pull request merged**